os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -1272,6 +1272,8 @@ _WEAK void wifi_set_mib(void)
 	wext_set_adaptivity(RTW_ADAPTIVITY_DISABLE);
 	//trp tis
 	wext_set_trp_tis(DISABLE);
+
+	rltk_wlan_enable_issue_deauth(ENABLE);

```

Comment:
Could you leave comment why you call this function in here?

Reply:
> Could you leave comment why you call this function in here?

@jeongchanKim , do I need to add comment before the line the code like "//enable issue deauth"?

Reply:
Yes. But not just `enable issue deauth`, but why. It is much better to understand.

Reply:
Added comment with explanation to the code. Please help to review.

Reply:
@keerthanalex Could you double-check this modification doesn't affect the compatibility with other APs?

Reply:
> @keerthanalex Could you double-check this modification doesn't affect the compatibility with other APs?

@davidfather, We tested this patch on 4 different AP's including the one's which doesn't have any compatibility issue.

Reply:
Could you inform us what kind of AP used?